index.css

```diff
@@ -0,0 +1,229 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-size: 14px;
+  font-family: arial, sans-serif;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  background-color: #fff;
+  position: relative;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row-reverse;
+  height: 8%;
+}
+
+.navbar ul {
+  font-size: 13px;
+  margin: 3px 10px 0px 0px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+li {
+  display: inline;
+  margin-right: 20px;
+}
+
+a {
+  text-decoration: none;
+  color: #000000;
+}
+
+img {
+  height: 24px;
+}
+
+.title {
+  height: 26%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: end;
+}
+
+.title img {
+  height: 52%;
+}
+
+.search {
+  height: 25%;
+}
+
+.language {
+  height: 35%;
+}
+
+.footer {
+  height: 13%;
+  background-color: #f2f2f2;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer_country {
+  height: 50%;
+  border-bottom: 1px solid #dadce0;
+}
+
+.footer_terms {
+  height: 50%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0px 5px;
+}
+
+.footer_text {
+  padding: 15px 30px;
+  border-bottom: 1px solid #dadce0;
+  font-size: 15px;
+  color: #70757a;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.footer_condition .footer_text li {
+  margin: 0px 0px 0px 30px;
+}
+
+.footer_condition .footer_text li a {
+  font-size: 14px;
+}
+
+.footer_about .footer_text li {
+  padding: 1px 10px 0px 0px;
+  font-size: 14px;
+}
+
+.footer_text li a {
+  color: #70757a;
+}
+
+.language_choices {
+  height: 20%;
+  width: fit-content;
+  margin: 1px auto;
+  display: flex;
+  flex-direction: row;
+}
+
+.language_choices li {
+  margin: 0px 5px;
+}
+
+.language_choices li a {
+  color: #1a0dab;
+  font-size: small;
+}
+
+.language_choices_text {
+  color: #4d5156;
```

Comment:
1. There is something called as CSS variable.
2. You can read about it from [here.](https://www.w3schools.com/css/css3_variables.asp)
3. Try to move all the colors value with CSS variables.